core/encoding/cli.go

```diff
@@ -77,6 +78,12 @@ func CLIFlags(envPrefix string) []cli.Flag {
 			Required: false,
 			EnvVar:   common.PrefixEnvVar(envPrefix, "PRELOAD_ENCODER"),
 		},
+		cli.StringFlag{
+			Name:     G2PowerOf2PathFlagName,
+			Usage:    "Path to G2 SRS points that are on power of 2. Either this flag or G2_PATH needs to be specified. For operator node, if both are specified, the node uses this flag first, if failed then tries to G2_PATH",
```

Comment:
"For operator node, if both are specified, the node uses this flag first", it should be using G2_PATH first?